mcr.go

```diff
@@ -19,7 +20,13 @@ type MCRService interface {
 	// CreatePrefixFilterList creates a Prefix Filter List on an MCR from the Megaport MCR API.
 	CreatePrefixFilterList(ctx context.Context, req *CreateMCRPrefixFilterListRequest) (*CreateMCRPrefixFilterListResponse, error)
 	// GetMCRPrefixFilterLists returns prefix filter lists for the specified MCR2 from the Megaport MCR API.
-	GetMCRPrefixFilterLists(ctx context.Context, mcrId string) ([]*PrefixFilterList, error)
```

Comment:
This shouldn't be changed as we've already put it in the V1 module and it would break compatibility.